app/screens/create_or_edit_channel/channel_info_form.tsx

```diff
@@ -390,18 +395,16 @@ export default function ChannelInfoForm({
                     </View>
                 </TouchableWithoutFeedback>
             </KeyboardAwareScrollView>
-            <View>
-                <Autocomplete
-                    position={autocompletePosition}
-                    updateValue={onHeaderChange}
-                    cursorPosition={header.length}
-                    value={header}
-                    nestedScrollEnabled={true}
-                    availableSpace={autocompleteAvailableSpace}
-                    inPost={false}
-                    growDown={!growUp}
-                />
-            </View>
+            <Autocomplete
+                position={animatedAutocompletePosition}
+                updateValue={onHeaderChange}
+                cursorPosition={header.length}
+                value={header}
+                nestedScrollEnabled={true}
+                availableSpace={animatedAutocompleteAvailableSpace}
+                inPost={false}
+                growDown={!growUp}
```

Comment:
I'd suggest changing the `growUp` var to `growDown` to avoid passing `!growUp`.  Or change the `growDown` prop and logic in `Autocomplete` to `growUp`